vnext/Desktop.IntegrationTests/WebSocketJSExecutorIntegrationTest.cpp

```diff
@@ -86,14 +91,15 @@ TEST_CLASS (WebSocketJSExecutorIntegrationTest) {
 
     string errorMessage;
     auto errorCallback = [&errorMessage](string message) { errorMessage = message; };
-    bool connected = jse->ConnectAsync("ws://localhost:8081/debugger-proxy?role=client", move(errorCallback)).get();
+    auto noop = []() {};
+    jse->ConnectAsync("ws://localhost:8081/debugger-proxy?role=client", move(errorCallback), noop, noop).get();
     // Point to a non-existing path.
     auto bigString = unique_ptr<JSBigString>(new JSBigStdString("http://localhost:8081/showme404"));
     jse->loadApplicationScript(std::move(bigString), "");
 
     jsThread->quitSynchronous();
 
-    Assert::IsTrue(connected);
+    // Assert::IsTrue(connected);
```

Comment:
Do you check in commented code?

Reply:
No we don't.  I deleted the whole file instead. (They were dead tests)